test/suites/projects.sh

```diff
@@ -1,12 +1,12 @@
 # Use the default project.
 test_projects_default() {
   # The default project is used by the default profile
-  lxc project show default | grep -q "/1.0/profiles/default$"
+  lxc project show default | grep -q "/1.0/profiles/default?project=default$"
```

Comment:
I don't think the api output should be changing here. I don't think we have historically included the default project on the URLs in the used by list.